src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -792,10 +1042,21 @@ subroutine decomp_rate_constants_bgc(bounds, num_soilc, filter_soilc, &
             spinup_geogterm_s1(c) = 1._r8
             spinup_geogterm_s2(c) = 1._r8
             spinup_geogterm_s3(c) = 1._r8
+            spinup_geogterm_m1(c) = 1._r8
+            spinup_geogterm_m2(c) = 1._r8
          end do
       endif
 
       !--- time dependent coefficients-----!
+      ! TODO slevis: Not immediately obvious how to reduce the if-else
+      !      repetition, bc t_scalar, w_scalar, o_scalar sum all j
+      !      to layer 1 in the if, while they do not in the else.
```

Comment:
@wwieder you mentioned that we would use w_scalar and o_scalar but didn't mention t_scalar. Regardless of whether we reduce the if-else repetition, I wanted to go over with you what we're keeping and what we're discarding from this section.

Reply:
...so pls ignore for now the next few lines of comments.

Reply:
@wwieder wrote by email "we won't use t_scalar in mimics."